pkg/serverless/otlp/otlp.go

```diff
@@ -31,7 +32,7 @@ type ServerlessOTLPAgent struct {
 // NewServerlessOTLPAgent creates a new ServerlessOTLPAgent with the correct
 // otel pipeline.
 func NewServerlessOTLPAgent(serializer serializer.MetricSerializer, tagger tagger.Component) *ServerlessOTLPAgent {
-	pipeline, err := coreOtlp.NewPipelineFromAgentConfig(pkgconfigsetup.Datadog(), serializer, nil, tagger)
+	pipeline, err := coreOtlp.NewPipelineFromAgentConfig(pkgconfigsetup.Datadog(), serializer, nil, tagger, hostnameimpl.NewHostnameService())
```

Comment:
Just so I understand, this was already being used by `NewPipelineFromAgentConfig` through the global hostname package, therefore, in Serverless, this should still keep acting as a Noop, right?